psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -2267,11 +2272,13 @@ PopulationAnalysisCalc::compute_mbis_multipoles(bool print_output) {
     // => Post-Processing <= //
  
     // Atomic density, as defined in Equation 5 of Verstraelen et al. (Negative to account for negative charge of electron)
```

Comment:
Update this comment, since you falsified it.

Reply:
@JonathonMisiewicz In the near future, I intend to add the BSISA charge partitioning scheme, which may potentially require the grid, etc. to calculate Hirshfeld properties